tests/acceptance/tests/Account/CustomerCredentials.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test('As a customer, I must be able to change email via account.', { tag: '@Account' }, async ({
```

Comment:
What about the negativ test? So ein mean that it is not possible to change the email with a wrong email without a @